tests/ui/target-feature/wasm-relaxed-simd.rs

```diff
@@ -0,0 +1,10 @@
+//@ only-wasm32
+//@ compile-flags: -Ctarget-feature=+relaxed-simd --crate-type=lib
+//@ build-pass
+#![feature(stdarch_wasm_relaxed_simd)]
+
+use std::arch::wasm32::*;
+
+fn test(a: v128, b: v128, m: v128) -> v128 {
```

Comment:
If `--crate-type=lib` then I think this needs to be `pub` otherwise it's just dead code.

Reply:
Done.